src/content/docs/new-relic-solutions/new-relic-solutions/cloud-adoption/modern-cloud-services.mdx

```diff
@@ -58,7 +58,7 @@ To get started configuring cloud service integrations, link your cloud service p
   >
     In this default dashboard for the [AWS EC2 monitoring integration](/docs/integrations/amazon-integrations/aws-integrations-list/aws-ec2-monitoring-integration), New Relic captures metrics for EC2 instances per region, instance state, and instance type. The dashboard also shows inventory for different software packages and configurations that are installed on those instances.
 
-    ![AWS EC2 dashboard example](./images/AWSEC2-Dashboard.png "AWS EC2 dashboard example")
+    ![AWS EC2 dashboard example](https://docs-dev.newrelic.com/sites/default/files/styles/inline_660px/public/thumbnails/image/AWSEC2-Dashboard.png?itok=79i3HTwf "AWS EC2 dashboard example")
```

Comment:
not sure why the image codemod would not run on this one in particular.

Reply:
perhaps because it's inside Collapser

Reply:
@jerelmiller can you think of anything that changed recently with codemods that may cause this?

Reply:
That shouldn't matter, all images should be changed regardless. Likely this image errored when downloading. It doesn't switch over the URL unless we've successfully downloaded the image.

Reply:
That being said, I did make one small change that fixed the "uncaught promise rejection" error that we'd see from time to time. Previously, if that uncaught promise rejection happened, it would more or less fail the job. Now there is a warning shown instead.

Reply:
gotcha. okay that makes sense. and that lines up with what i've experienced intermittently with images when running migration. they seem to sometimes time out (saw the error about socket hang up). maybe Acquia server limiting / not able to serve so many image file requests at once? i've run it 3 times now and it still seems to fail some of these image downloads.